src/main/java/edu/byu/cs/util/DateTimeUtils.java

```diff
@@ -31,9 +31,6 @@ public static String getDateString(long timestamp, boolean includeTime) {
      * @return the number of days late or 0 if the submission is not late
      */
     public static int getNumDaysLate(ZonedDateTime handInDate, ZonedDateTime dueDate) {
-        // end of day
-        dueDate = dueDate.withHour(23).withMinute(59).withSecond(59);
-
```

Comment:
Thanks for remembering this from the TA meeting; I forgot to follow through with it. We might have a problem with at the current moment though, as it looks like the due dates on Canvas are still set to 8pm
![image](https://github.com/softwareconstruction240/autograder/assets/10139415/f69dbc23-2118-407e-a0c6-7d409d255b26)

Reply:
I'm definitely not opposed to letting Canvas be the source of truth, but we have quite publicly promised that students have until midnight, so I don't want to include this change until Canvas is correct

Reply:
I'm definitely not opposed to letting Canvas be the source of truth, but we have publicly promised that students have until midnight so I don't want to include this change until Canvas is correct